hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/evm/config/EvmConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package com.hedera.mirror.web3.evm.config;
+
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.github.benmanes.caffeine.cache.Caffeine;
+import java.util.concurrent.TimeUnit;
+import org.springframework.cache.CacheManager;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cache.caffeine.CaffeineCacheManager;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableCaching
+public class EvmConfiguration {
+
+    public static final String CACHE_MANAGER_10M = "10M";
+    public static final String CACHE_MANAGER_500NS = "500NS";
+
+    @Bean
+    Caffeine caffeineConfig() {
+        return Caffeine.newBuilder();
+    }
+
+    @Bean(CACHE_MANAGER_10M)
+    public CacheManager cacheManager10M(Caffeine caffeine) {
+        CaffeineCacheManager caffeineCacheManager = new CaffeineCacheManager();
+        caffeine.expireAfterAccess(10, TimeUnit.MINUTES).maximumSize(10000);
+        caffeineCacheManager.setCaffeine(caffeine);
```

Comment:
You can't share the `Caffeine` instance between two different managers. Right now both will have the same config.

```suggestion
   CacheManager cacheManager10M() {
        var caffeine = Caffeine.newBuilder().expireAfterAccess(10, TimeUnit.MINUTES).maximumSize(10000);
        CaffeineCacheManager caffeineCacheManager = new CaffeineCacheManager();
        caffeineCacheManager.setCaffeine(caffeine);
```